document/js-api/index.bs

```diff
@@ -729,9 +729,20 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
 
 <h3 id="globals">Globals</h3>
 
+<pre class="idl">
+enum WebAssemblyValueType {
```

Comment:
I'm still confused why we prefix (only) this IDL type with "WebAssembly", that seems inconsistent. Perhaps remove?

Reply:
Good point, I forgot to ask if we should rather remove it.